ArduCopter/land_detector.cpp

```diff
@@ -196,7 +196,7 @@ void Copter::update_throttle_mix()
         // check if landing
         const bool landing = flightmode->is_landing();
 
-        if ((large_angle_request && !landing) || large_angle_error || accel_moving || descent_not_demanded) {
+        if (((large_angle_request || gps_air_mode == AirMode::AIRMODE_DISABLED) && !landing) || large_angle_error || accel_moving || descent_not_demanded) {
```

Comment:
This logic is not consistent with the comment on the AUX function. This says only disable if you are not landing, the comment says disable event if you *are* landing

Reply:
correct thanks!